lib/rails_stdout_logging/rails.rb

```diff
@@ -1,5 +1,5 @@
 module RailsStdoutLogging
-  class StdoutLogger < ::Logger
+  class StdoutLogger < ::ActiveSupport::Logger
```

Comment:
@schneems 3.x [reopens the stdlib Logger class](https://github.com/rails/rails/blob/3-2-stable/activesupport/lib/active_support/core_ext/logger.rb#L33-L41) to add silencing.

It was deprecated before 4.x, and brought back as a separate module included in AS Logger.